xikolo-common/assets-common.xcassets/welcome-globe.imageset/Contents.json

```diff
@@ -1,24 +0,0 @@
-{
-  "images" : [
-    {
-      "idiom" : "universal",
-      "filename" : "globus.png",
-      "scale" : "1x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "2x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "3x"
-    }
-  ],
-  "info" : {
-    "version" : 1,
-    "author" : "xcode"
-  },
-  "properties" : {
-    "template-rendering-intent" : "original"
-  }
-}
```

Comment:
If you remove the welcome images you need to fix all Storyboards and remove the ImageView's default images (e.g. the ImageView of CourseCell in the tvOS Storyboard).